inductiva/_cli/resources.py

```diff
@@ -9,12 +9,63 @@ def list_resources(args):
     inductiva.resources.storage.get_space_used()
 
 
+def list_resources_available(args):
+    """List all available machines"""
+    del args  # unused
+
+    machines_dict = {
+        "c2-standard-": [4, 8, 16, 30, 60],
+        "c3-standard-": [4, 8, 22, 44, 88, 176],
+        "c2d-standard-": [2, 4, 8, 16, 32, 56, 112],
+        "c2d-highcpu-": [2, 4, 8, 16, 32, 56, 112],
+        "e2-standard-": [2, 4, 8, 16, 32],
+        "n2-standard-": [2, 4, 8, 16, 32, 48, 64, 80, 96, 128],
+        "n2d-standard-": [2, 4, 8, 16, 32, 48, 64, 80, 96, 128, 224],
+        "n1-standard-": [1, 2, 4, 8, 16, 32, 64, 96]
+    }
+
+    print("Available machine types\n")
+    print("machine-type: [cores-available]")
+    for machine_type, cores in machines_dict.items():
+        cores_str = ", ".join(str(core) for core in cores)
+        print(f"{machine_type}: [{cores_str}]")
+
+    print("\n E.g. of machine: c2-standard-8\n")
+
+
+def terminate_machine(args):
+    """Terminate a machine with the given name."""
+    machine_name = args.machine_name
+
+    print("Terminating machine... If exists.")
+    machines_list = inductiva.resources.machine_groups.get()
+
+    for machine in machines_list:
```

Comment:
This is one way of doing it, or if we want to avoid iterating over the machines and retrieving all of them, you could use [this](https://github.com/inductiva/inductiva/blob/fd8a213786328fa8c939e45ec1526aadb160fa9e/inductiva/resources/resources.py#L33) that I initially wrote to retrieve a single machine by it's name. The endpoint is still there so you could just implement it here in the client side.

Reply:
The problem is that I think there is no arg/flag in the backend to differentiate between elastic or non-elastic.
Given that the purpose is to just kill the machine, I will keep it this way and in the future we can re-think the whole machine_groups with the idea of coupling things to it.